README.md

```diff
@@ -66,6 +66,14 @@ tile_bytes = slide.read_encoded_tile(6, (1, 1))
 slide.close()
 ```
 
+## Settings
```

Comment:
Add way to turn this on/off on commandline?

What is the best way, should it be default strict or default loose? I would say

- strict: Require that everything is perfect, strict-mode is turned on. Maybe what we require in BigPicture should match strict mode, and can be used to validate files.
- normal: Anything that is not required for the library to work well should be accepted. Major performance issues is still warned on. Normal should be the default, if the library can open the file, I as a end-user want it just to be opened and not bother about any special settings.

Reply:
Changed default to non-strict settings. Need to address presence or not of frame of reference in future.